TestResultSummaryService/utils/BenchmarkMathCalculation.js

```diff
@@ -0,0 +1,84 @@
+const math = require("mathjs");
```

Comment:
The original issue is fixed and the file is delivered https://github.com/adoptium/aqa-test-tools/blob/master/TestResultSummaryService/perf/BenchmarkMathCalculation.js. 

We do not need this file and [package-lock.json](https://github.com/adoptium/aqa-test-tools/pull/1030/files#diff-24f382b7b9e9a6e5d13c964969afc53119ee96dfb1355dec2db387bf355ec6fe) below.